src/test/scala-2.12/org/apache/james/gatling/simulation/jmap/rfc8621/WebsocketSimpleSimulation.scala

```diff
@@ -0,0 +1,26 @@
+package org.apache.james.gatling.simulation.jmap.rfc8621
+
+import io.gatling.core.Predef._
+import io.gatling.core.scenario.Simulation
+import org.apache.james.gatling.control.{Password, User, UserFeeder, Username}
+import org.apache.james.gatling.jmap.rfc8621.scenari.WebsocketSimpleScenario
+import org.apache.james.gatling.simulation.{Configuration, HttpSettings}
+
+class WebsocketSimpleSimulation  extends Simulation {
```

Comment:
IMO we do not even need the simulation now...

Reply:
I have no issue of taking it out :)